modo/helpers.py

```diff
@@ -23,13 +24,79 @@ def dict_to_instance(element: Mapping[str, Any]) -> Any:
     )
 
 
+def set_part_of_relationship(
```

Comment:
```suggestion
def set_haspart_relationship(
```
**suggestion:** for consistency, we may want to use `haspart` instead of `partof`, as this matches what the properties we're adding (`has_sample`, `has_assay`, ...)